dockerizeit/master/permissions.properties

```diff
@@ -0,0 +1,12 @@
+user1 {
+		userId = hudson.security.ACL.ANONYMOUS_USERNAME
+		permissions = [
+			"hudson.model.Computer.Create",
```

Comment:
Skip that - you don't need to import them if you specify the full path to the class. Late night commenting is no good

What I mean is we should do it as it is done for the slaves - look for mode and retention

https://github.com/Praqma/JenkinsAsCodeReference/blob/master/dockerizeit/master/README.md#slaves

Also, do you consider 'Issue66' as a good commit message? Or what is the thinking behind that?
Where is documentation update